src/view/com/auth/SplashScreen.tsx

```diff
@@ -17,9 +28,32 @@ export const SplashScreen = ({
 }) => {
   const pal = usePalette('default')
   const {_} = useLingui()
+  const {showLoggedOut} = useLoggedOutView()
+  const {setShowLoggedOut} = useLoggedOutViewControls()
```

Comment:
Seems wrong that `SplashScreen` "knows" about these if the logic to show the splash screen _for this particular reason_ is inside `withAuthRequired`. I suggest that the `SplashScreen` takes an `onDismiss` prop. If `onDismiss != null`, show the close button which calls `onDismiss`. Then we can hoist the "show logged out" logic somewhere above — maybe to `withAuthRequired` (which can pass it down to `LoggedOut` -> `SplashScreen`).